surfactant/filetypeid/id_magic.py

```diff
@@ -208,6 +208,36 @@ def identify_file_type(filepath: str) -> Optional[str]:
                 if cm == 8:
                     if (cmf * 256 + flg) % 31 == 0:
                         return "ZLIB"
+            # cpio:
+            # https://www.systutorials.com/docs/linux/man/5-cpio/
+            if int.from_bytes(magic_bytes[:2], byteorder="big", signed=False) == 0o70707:
+                return "CPIO_BIN big"
+            if int.from_bytes(magic_bytes[:2], byteorder="little", signed=False) == 0o70707:
+                return "CPIO_BIN little"
+            if magic_bytes[:6] == b"070707":
+                return "CPIO_CHAR_OLD"
+            if magic_bytes[:6] == b"070701":
+                return "CPIO_ASCII_NEW"
+            # zstd:
+            # https://datatracker.ietf.org/doc/html/rfc8878
+            if magic_bytes[:4] == b"\x28\xb5\x2f\xfd":
+                return "ZSTANDARD"
+            if magic_bytes[:4] == b"\x37\xa4\x30\xec":
+                return "ZSTANDARD_DICTIONARY"
+            # iso:
+            # https://www.garykessler.net/library/file_sigs.html
```

Comment:
```suggestion
            # https://en.wikipedia.org/wiki/List_of_file_signatures
```

The wikipedia page has the same info in its table -- garykessler's site seems to be gone now.